Jenkinsfile

```diff
@@ -46,14 +46,15 @@ pipeline {
 					agent {
 						label 'docker'
 					}
-					environment {
-						GITHUB_API = credentials('github-token')
```

Comment:
tbh you didn't need to change anything other than the credential id

Reply:
Oh, good to know :)
Will try again here or on other similar cases.